src/router/index.tsx

```diff
@@ -28,7 +29,7 @@ export const AppRouter = () => {
       <Route exact path="/auctions/:auctionId" component={AuctionView} />
       <Route exact path="/auctions/demo" component={SimulationView} />
       <Route exact path="/auctions/fixed/:auctionId" component={FixedPriceAuctionView} />
-      <Route exact path="/about" component={AuctionsView} />
+      <Route exact path="/about" component={StaticView} />
```

Comment:
If the goal is to create a static view with dynamic content, adjust the route as follows

```typescript
<Route exact path="/:staticId" component={StaticView} />
```

Since `staticId` is anticipated in `StaticView`

Reply:
jep, staticId is there, but not used right now